crates/sui-genesis-builder/src/lib.rs

```diff
@@ -1135,7 +1135,8 @@ pub fn generate_genesis_system_object(
             )?;
         }
 
-        if protocol_config.enable_accumulators() {
+        if protocol_config.enable_accumulators() && protocol_config.create_root_accumulator_object()
```

Comment:
Probably not that important, but semantically this should only depend on create_root_accumulator_object, not enable_accumulators anymore.

Reply:
It should, but this actually made the tests much more convenient to write (since we start without the accumulator object this way)

Reply:
I'm a bit confused though, isn't the whole point of doing a separate config so that we can enable `create_root_accumulator_object` without having to enable `enable_accumulators`